cli/render.go

```diff
@@ -64,16 +202,17 @@ func (c *RenderCommand) Run(args []string) int {
 		return 1
 	}
 
-	d := glint.New()
+	var renders = []Render{}
+
+	// Iterate the rendered files and add these to the list of renders to
+	// output. This allows errors to surface and end things without emitting
+	// partial output and then erroring out.
 
-	// Iterate the rendered files and add these to the Glint document.
-	// TODO(jrasell): trim at least the templates directory name from the name
-	//  as it doesn't provide much benefit.
 	for name, renderedFile := range renderOutput.DependentRenders() {
-		addRenderToDoc(d, name, renderedFile)
+		renders = append(renders, Render{Name: formatRenderName(name), Content: renderedFile})
```

Comment:
How do you feel about adding these Names to the ErrorContext? Same for ParentRenders.